security/src/main/java/io/micronaut/security/token/propagation/AbstractOutgoingRequestProcessorMatcher.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.security.token.propagation;
+
+import io.micronaut.http.util.OutgointRequestProcessorMatcher;
+
+import java.util.regex.Pattern;
+
+/**
+ * Base implementation class for {@link OutgointRequestProcessorMatcher}.
+ *
+ * @author Álvaro Sánchez-Mariscal
+ */
+public abstract class AbstractOutgoingRequestProcessorMatcher implements OutgointRequestProcessorMatcher {
+
+    protected String serviceIdRegex;
```

Comment:
misses nullability annotation.

Reply:
Nope, see the other comment. Will add it to the accessors.

Also, the previous code didn't have any nullability annotations. Assuming `@Nullable`